api/entry.go

```diff
@@ -15,6 +15,7 @@ import (
 	"miniflux.app/model"
 	"miniflux.app/storage"
 	"miniflux.app/validator"
+	"miniflux.app/reader/processor"
```

Comment:
Looks like there is a linter issue here (see GitHub Actions logs).

Reply:
`gofmt -s -w ` sorted the import for me, it should be ok now